cmake/SetupArpack.cmake

```diff
@@ -0,0 +1,22 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+find_package(Arpack REQUIRED)
+
+add_library(Arpack INTERFACE IMPORTED)
+
+if(ARPACK_FOUND)
+  message(STATUS "Arpack libs: " ${ARPACK_LIBRARIES})
+else()
+  message(FATAL_ERROR "Arpack not found\n")
+endif()
+
+set_property(TARGET Arpack
+  APPEND PROPERTY INTERFACE_LINK_LIBRARIES ${ARPACK_LIBRARIES})
+set_property(TARGET Arpack
+  APPEND PROPERTY INTERFACE_LINK_OPTIONS ${ARPACK_LINKER_FLAGS})
```

Comment:
You aren't getting these linker flags from anywhere, so the variable is empty and this commend doesn't do anything. Are they needed?